java/src/com/ctre/phoenix/sensors/PigeonIMU.java

```diff
@@ -601,7 +601,7 @@ public ErrorCode getRawGyro(double[] xyz_dps) {
 	/**
 	 * Get Accelerometer tilt angles.
 	 *
-	 * @param tiltAngles Array to fill with x[0], y[1], and z[2] angles.
+	 * @param tiltAngles Array to fill with x[0], y[1], and z[2] angles in degrees.
```

Comment:
I don't actually know what the units are. It would be good to not have to assume